backoffice/pages/catalog/categories/create.tsx

```diff
@@ -74,12 +74,8 @@ const CategoryCreate: NextPage = () => {
                   slug?.setAttribute(
                     'value',
                     slugify(e.target.value, {
-                      replacement: '-', // replace spaces with replacement character, defaults to `-`
-                      remove: undefined, // remove characters that match regex, defaults to `undefined`
-                      lower: true, // convert to lower case, defaults to `false`
-                      strict: false, // strip special characters except replacement, defaults to `false`
-                      locale: 'vi', // language code of the locale to use
-                      trim: true, // trim leading and trailing replacement chars, defaults to `true`
+                      lower: true,
+                      strict: false,
```

Comment:
why strict is false here while others are true ^^

Reply:
sorry, my mistake